tracket-react/src/components/map/Map.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  MapContainer,
+  TileLayer,
+  Marker,
+  Tooltip,
+} from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+import {
+  LAYER_ATTRIBUTION,
+  LAYER_URL,
+  MAP_CENTER_LAT,
+  MAP_CENTER_LON,
+  ZOOM,
+} from "../../config";
+import MarkerClusterGroup from "react-leaflet-cluster";
+import { useEffect, useState } from "react";
+import { getLocations } from "../api/locations";
+import { type Location } from "../../types/api";
+import { type LatLngTuple } from "leaflet";
+
+const Map = () => {
+  const [locations, setLocations] = useState<Location[]>([]);
+
+  useEffect(() => {
+    const fetchLocations = async () => {
+      try {
+        const result = await getLocations();
+        setLocations(result.locations);
+      } catch (err: unknown) {
+        console.error(err);
+      }
+    };
+
+    fetchLocations();
+  }, []);
+
+  return (
+    <MapContainer center={[MAP_CENTER_LAT, MAP_CENTER_LON]} zoom={ZOOM}  zoomControl={true}>
+      <TileLayer url={LAYER_URL} attribution={LAYER_ATTRIBUTION} />
+
+      <MarkerClusterGroup>
+        {locations.map((l) => {
+          const position: LatLngTuple = [l.latitude, l.longitude];
+          const toolTipActive = l.active
+            ? "Active Location"
```

Comment:
Similar to v1: https://github.com/CivicTechTO/tRacket-dashboard/blob/6a0a3c82ba72210d81b8b0edde61858039cb70f9/app/src/app_components.py#L192-L207